### ISSUE TITLE
Create calendar for EEW Project?

### ISSUE THREAD

**Issue by Frijol** [edgi-govdata-archiving/EEW_Planning#88]
Could go in readme of this repo. Purpose is to keep us all informed. Would be nice to list:

* Any upcoming events
* When @saraannwylie 's course buyout starts/ends
* When @lourdesvera @calder @crgreenleaf official employment starts/ends
* Any events we are considering or milestone dates

┆Issue is synchronized with this [Trello card](https://trello.com/c/Vf6GaTAu)

**Comment by lourdesvera** [comment]
orr as a google calendar?

**Comment by lightandluck** [comment]
Oops, sorry, was just checking things out in the project board. Thought I was just closing the tab, not closing the issue itself!

**Comment by lourdesvera** [comment]
@Frijol @saraannwylie do you think a google calendar would be good?

**Comment by Frijol** [comment]
seems reasonable to me! Only thing I'd want to be clear on is: is this an internal calendar w/hire dates? Or a public calendar of events? Or do we want both?

cc @EDGI-Comms would play nicely with a Wordpress plugin for our site

**Comment by EDGI-Comms** [comment]
Agree! And we should definitely at least have an external public events calendar.

**Comment by EDGI-Comms** [comment]
Maybe @calderEDJ can research Google calendar plugins?

**Comment by lourdesvera** [comment]
We have a nice calendar plugin, so we can put the events on one calendar and internal hiring calendar on another.

**Comment by lourdesvera** [comment]
EEW planning: https://calendar.google.com/calendar?cid=czNkcmltNzRnbG05dWtsY2E4ZjlqNnZhaGdAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ

EEW events: https://calendar.google.com/calendar?cid=c280bXJsNjk5YWVhNTE5bnQxNzhwNTBwMzhAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ

**Comment by saraannwylie** [comment]
draft ideas for timeline milestone for the summer in slide 13 here. [](https://docs.google.com/presentation/d/1hYUa9-09a4VlG2gEtXguSsry8Z82xQy5nwLCoi0cclA/edit#slide=id.g85c9bd05a9_1_32)
